assets/src/blocks/ENForm/ENFormInPlaceEdit.js

```diff
@@ -0,0 +1,409 @@
+    switch (style) {
+      case 'side-style':
+        return 'block-header block-wide';
+      case 'full-width-bg':
+        return 'block-footer block-wide';
+      default:
+        return '';
+    }
+  })(en_form_style);
+
+  return (
+    <>
+    <BlockControls>
+      <ToolbarGroup
+        isCollapsed={ true }
+        icon={ activeTpl.icon }
+        label={ activeTpl.title }
+        controls={
+          templates.map((tpl) => {
+            return {
+              icon: tpl.icon,
+              title: tpl.title,
+              isActive: activeTplId === tpl.id,
+              onClick: () => { setActiveTplId(tpl.id) }
+            }
+          })
+        }
+      />
+    </BlockControls>
+
+    <section 
+      className={`block enform-wrap enform-${en_form_style} ${section_style}`}
+    >
+      {style_has_image &&
+        <BackgroundImage {...{attributes}} />
+      }
+      <div className="container">
+        <div className="row">
+          <div className="col-md-12">
+            {is_side_style &&
+              <SideContent {...{attributes, setAttributes}} />
+            }
+            {activeTplId === 'signup' &&
+              <Signup {...{attributes, setAttributes}} />
+            }
+            {activeTplId === 'thankyou' &&
+              <ThankYou {...{attributes, setAttributes}} />
+            }
+          </div>
+        </div>
+      </div>
+    </section>
+    </>
+  )
+}
+
+const BackgroundImage = ({attributes}) => {
+  const { 
+    background,
+    background_image_src,
+    background_image_srcset,
+    background_image_sizes,
+    background_image_focus,
+  } = attributes;
+
+  if (! background) {
+    return null;
+  }
+
+  return (
+    <picture>
+      <img src={ background_image_src || ''}
+        style={{objectPosition: background_image_focus || {}}}
+        border="0"
+        srcSet={background_image_srcset || ''}
+        sizes={background_image_sizes || ''}
+        className={ background > 0 ? `wp-image-${background}` : ''}
+      />
+    </picture>
+  );
+}
+
+const SideContent = ({attributes, setAttributes}) => {
+  const { 
+    content_title,
+    content_description,
+    content_title_size,
+    campaign_logo,
+  } = attributes;
+
+  return (
+    <>
+    <BlockControls>
+      <ToolbarGroup
+        isCollapsed={ true }
+        icon="heading"
+        label={content_title_size.toUpperCase()}
+        controls={
+          ['h1', 'h2'].map((size) => {
+            const isActive = content_title_size === size;
+            return {
+              isActive,
+              icon: "heading",
+              title: size.toUpperCase(),
+              onClick: () => { setAttributes({content_title_size: size}) }
+            }
+          })
+        }
+      />
+    </BlockControls>
+    <div className="form-caption">
+      {campaign_logo &&
+        <img src={ campaign_logo }
+            alt={ content_title }
+            className="campaign-logo" />
+      }
+      <RichText
```

Comment:
We've agreed with the design team that we should remove all formatting options from `RichText` components (more info [here](https://jira.greenpeace.org/browse/PLANET-5780)), could you please already do that with this block? 🙂